airbyte-integrations/connectors/source-captain-data/source_captain_data/schemas/job_results.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
```

Comment:
Please add `additionalProperties: true` to all streams. This make more easy to receive new fields.